.github/ISSUE_TEMPLATE/2_documentation.yml

```diff
@@ -0,0 +1,33 @@
+name: Documentation
+description: Create a documentation related issue.
+labels:
+  - type::documentation
+body:
+  - type: markdown
+    attributes:
+      value: |
+        > **Note**
+        Since there are already a lot of open issues, please also take a moment to search existing ones to see if your bug has already been reported. If you find something related, please upvote that issue and provide additional details as necessary.
+        💐 Thank you for helping to make Conda better. We would be unable to improve Conda without our community!
```

Comment:
```suggestion
        💐 Thank you for helping to make conda better. We would be unable to improve conda without our community!
```

Reply:
A lowercase c is what Jannis is suggesting here. But yeah, regardless of whether you're talking about the tool or organization, go lowercase unless you're starting a sentence or header with the word.